### PR TITLE
Drop python 3.5 support

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: generate_qfp.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def _uuid(identifier: str) -> str:
         # General info
         lines.append('(librepcb_package {}'.format(uuid_pkg))
         lines.append(' (name "{}")'.format(full_name))
-        lines.append(' (description "{}")'.format(full_description, generator))
+        lines.append(' (description "{}\\n\\nGenerated with {}")'.format(full_description, generator))
```
Comment (dbrgn) [inline generate_qfp.py:353]: This seems to be an unrelated change...

Comment (rnestler) [reply generate_qfp.py:353]: Read the commit message :wink: This fixes a flake8 error that the parameter `generator` wasn't used :slightly_smiling_face:

Comment (dbrgn) [reply generate_qfp.py:353]: Ah, makes sense!

**Review by dbrgn — APPROVED** [review]
